xline-client/src/types/kv.rs

```diff
@@ -674,3 +674,51 @@ impl From<Txn> for xlineapi::TxnRequest {
         txn.req
     }
 }
+
+/// Compaction Request compacts the key-value store up to a given revision.
+/// All superseded keys with a revision less than the compaction revision will be removed.
```

Comment:
Compaction is not a simple deletion. It won't remove the latest revisions of keys even if their revisions are less than the compaction revision.

Reply:
an extra note is added

Reply:
It's "Note that", not "Not that". You can use this as a comment: 
```suggestion
/// All keys with revisions less than the given revision will be compacted. The compaction process will remove all historical versions of these keys, except for the most recent one.  

/// For example, here is a revision list: [(A, 1), (A, 2), (A, 3), (A, 4), (A,5)]. We compact at revision 3. After the compaction, the revision list will become [ (A, 3), (A, 4), (A,5)]. All revisions less than 3 are deleted.  The latest revision, 3, will be kept. 
```

Reply:
Sorry for the typo. the comment is updated